pkgs/top-level/all-packages.nix

```
@@ -11490,7 +11490,7 @@ with pkgs;
   opentelemetry-collector = callPackage ../tools/misc/opentelemetry-collector { };
   opentelemetry-collector-contrib = callPackage ../tools/misc/opentelemetry-collector/contrib.nix { };
 
-  openvswitch-lts = callPackage ../by-name/op/openvswitch/lts.nix { };
+  openvswitch-lts = callPackage ../py-name/op/openvswitch/lts.nix { };
```

```suggestion
  openvswitch-lts = callPackage ../by-name/op/openvswitch/lts.nix { };
```
I guess this is a typo?